### PR TITLE
chore(docs): update the readme

### PR DESCRIPTION
**Pull request by kelset**
### Description

This PR is a follow up from a conversation we had last week about improving the state of the README, and add references to the migrating guide in it.

To make it shorter, I also moved the quick start guide to its [own wiki page](https://github.com/microsoft/react-native-test-app/wiki/Quick-Start). In the migration wiki I've also added a few more notes about certain things we had to do for blur (you can see them at the bottom of [this section](https://github.com/microsoft/react-native-test-app/wiki/Migrate-an-Existing-Test-App#configure-the-test-app)).

### Platforms affected

N/A

### Test plan

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
 ["Improve all the repos – exploring Microsoft’s DevExp"](https://youtu.be/DAEnPV78rQc?t=499)
-talk by @kelset and @tido64 from React Native Europe 2021.
+talk by [@kelset](https://github.com/kelset) and [@tido64](https://github.com/tido64) from React Native Europe 2021.
 
-For more about
+Read more about
 [the motivation](https://github.com/microsoft/react-native-test-app/wiki#motivation)
 and [the design](https://github.com/microsoft/react-native-test-app/wiki/Design)
-of this tool, you can refer to the wiki.
+of this tool.
```
Comment (tido64) [inline README.md]: Can we rephrase this so that it still makes sense even without the hyperlinks?

Comment (tido64) [reply README.md]: Same for all the new lines below. I like to have proper sentences for screen readers etc.

Comment (kelset) [reply README.md]: ok, I'll take care of it now - thanks, I didn't think about screen readers 👍

Comment (kelset) [reply README.md]: @tido64 check it out now, lmk if it's better

**Review by tido64 — APPROVED** [review]
Thanks, this looks a lot better ❤️

**Pull request merged**